src/Templates/Views/elements/textbox.pode

```diff
@@ -38,8 +38,15 @@ $(
             }
 
             $value = [string]::Empty
+            $textareavalue = [string]::Empty
             if (![string]::IsNullOrWhiteSpace($data.Value)) {
                 $value = "value='$($data.Value)'"
+                $textareavalue = "$($data.Value)"
+            }
+
+            $maxLength = [string]::Empty
+            if (![string]::IsNullOrWhiteSpace($data.maxLength)) {
```

Comment:
With `$data.MaxLength` being a integer, doing `$data.MaxLength -ge 0` would be safer; with the string check `-100` would be valid 😄 

It might also be worth putting validation into `New-PodeWebTextbox` to make sure the value is >=0